storage/src/adb/any/variable/mod.rs

```diff
@@ -720,17 +715,23 @@ impl<E: RStorage + Clock + Metrics, K: Array, V: Codec, H: CHasher, T: Translato
 
     /// Prune historical operations that are behind the inactivity floor. This does not affect the
     /// db's root or current snapshot.
-    pub(super) async fn prune_inactive(&mut self) -> Result<(), Error> {
+    pub async fn prune(&mut self, loc: Option<u64>) -> Result<(), Error> {
```

Comment:
I didn't realize that we exposed inactivity_floor, will make this not an option.

Reply:
one sec